packages/vitest/src/node/core.ts

```diff
@@ -364,13 +364,13 @@ export class Vitest {
   }
 
   async changeNamePattern(pattern: string, files: string[] = this.state.getFilepaths(), trigger?: string) {
-    this.config.testNamePattern = pattern ? new RegExp(pattern) : undefined
+    this.config.testNamePattern = pattern.trim() ? new RegExp(pattern) : undefined
```

Comment:
Shouldn't trimmed patter be used in `new RegExp`? Also maybe it's better to trim it before `changeNamePattern` is called? The same for `changeFilenamePattern`

Reply:
Maybe here?
https://github.com/vitest-dev/vitest/blob/d6225764856dd62cb1b21ec5bf4619052609c244/packages/vitest/src/node/stdin.ts#L98-L99

Reply:
Yes

Reply:
Updated.